src/util/jira-utils.ts

```diff
@@ -57,3 +58,36 @@ interface Author {
 	};
 }
 
+/**
+ * Parses strings for Jira issue keys for commit messages,
+ * branches, and pull requests.
+ *
+ * Accepted patterns:
```

Comment:
This would be really useful on the Readme too🤔

Reply:
Eh, seems a bit much to put it in the readme.  We'll just be duplicating the documentation for this function and it seems to be a very specific implementation that most won't care about until they change the function itself.